assignment09/part01/process.php

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<head>
+    <title>Assignment 9, Part 1</title>
+</head>
+<body>
+
+<?php
+    $weight = $_POST['weight'];
+    $height = $_POST['height'];
+
+    //formula for calculating BMI:
+    //BMI = (Weight in Pounds / (Height in inches x Height in inches)) x 703
+    //source: http://www.bmi-calculator.net/bmi-formula.php
+
```

Comment:
this is where you would calculate BMI and have your if/else for underweight/normal/obese.